dspace-api/src/main/java/org/dspace/administer/StructBuilder.java

```diff
@@ -132,22 +152,41 @@ public static void main(String[] argv)
         }
 
         if (output == null || eperson == null || file == null) {
-            usage();
-            System.exit(0);
+            usage(options);
+            System.exit(1);
         }
 
         // create a context
         Context context = new Context();
 
         // set the context
-        context.setCurrentUser(ePersonService.findByEmail(context, eperson));
+        try {
+            context.setCurrentUser(ePersonService.findByEmail(context, eperson));
+        } catch (SQLException ex) {
+            System.err.format("That user could not be found:  %s%n", ex.getMessage());
```

Comment:
For all these new error messages, I wonder if it'd be worth printing the entire stacktrace to `System.err` instead of *just* the Message.  Remember, `getMessage()` can return null, so I'm worried that there's some scenarios where this could make the error output less useful.

Reply:
I rather dislike spewing a stack trace on the console.  We could log it.
In this case, if getMessage() returns null then we write to the console "That user could not be found:  null".  "That user" is the value of the -e option.
If Exception.getMessage() ever *does* return null, we should file a bug report.

Reply:
OK, good enough for me. I'll mark this as resolved. I agree that stacktraces to console are not ideal, which is why I didn't feel strongly either way here. So, as you said, let's leave it as-is and fix specific scenarios as needed in the future.